Tests/MeshLib/TestMeshNodeSearch.cpp

```diff
@@ -60,6 +61,64 @@ class MeshLibMeshNodeSearchInSimpleHexMesh : public testing::Test
 	Mesh* _hex_mesh;
 };
 
+TEST_F(MeshLibMeshNodeSearchInSimpleQuadMesh, PointSearch2)
+{
+	double _dx = _geometric_size / _number_of_subdivisions_per_direction;
+	double dx_half = _dx*0.5;
+
+	ASSERT_TRUE(_quad_mesh != nullptr);
+
+	// 2 perform search and compare results with expected vals
+	MeshGeoToolsLib::SearchLength search_length(dx_half);
+	MeshGeoToolsLib::MeshNodeSearcher mesh_node_searcher(*_quad_mesh,
+		search_length);
+
+	{
+		GeoLib::Point pnt(0.0, 0.0, 0.0);
+		std::vector<std::size_t> id_vec(mesh_node_searcher.getMeshNodeIDsForPoint(pnt));
+		EXPECT_EQ(1u, id_vec.size());
+		EXPECT_EQ(0u, mesh_node_searcher.getMeshNodeIDsForPoint(pnt)[0]);
+	}
+	{
+		GeoLib::Point pnt(dx_half*0.99, 0.0, 0.0);
+		EXPECT_EQ(1u, mesh_node_searcher.getMeshNodeIDsForPoint(pnt).size());
+		EXPECT_EQ(0u, mesh_node_searcher.getMeshNodeIDsForPoint(pnt)[0]);
+	}
+	{
+		GeoLib::Point p(dx_half, 0.0, 0.0);
+		EXPECT_EQ(0u, mesh_node_searcher.getMeshNodeIDsForPoint(p).size());
+	}
+	{
+		GeoLib::Point p(dx_half*1.01, 0.0, 0.0);
+		ASSERT_EQ(1u, mesh_node_searcher.getMeshNodeIDsForPoint(p).size());
+		ASSERT_EQ(1u, mesh_node_searcher.getMeshNodeIDsForPoint(p)[0]);
+	}
+}
+
+/*
+TEST_F(MeshLibMeshNodeSearchInSimpleQuadMesh, PointSearch3)
```

Comment:
Forgot to mention this one. Either add or remove or at least comment why commented.